src/workerd/jsg/wrappable.c++

```diff
@@ -46,6 +46,11 @@ void HeapTracer::clearWrappers() {
   clearFreelistedShims();
 }
 
+using JSGWrappable = workerd::jsg::Wrappable;
+#if (V8_MAJOR_VERSION >= 14 && V8_MINOR_VERSION >= 1)
```

Comment:
You need to check for (major == 14 && minor >= 1) || major > 14 otherwise 15.0 and 16.0 are not covered.